packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -906,9 +906,8 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
             child: widget.underline ?? Container(
               height: 1.0,
               decoration: const BoxDecoration(
-                border: Border(bottom: BorderSide(color: Color(0xFFBDBDBD), width: 0.0))
+                border: Border(bottom: BorderSide(color: Color(0xFFBDBDBD), width: 0.0))),
```

Comment:
nit (this is slightly more readable to me): 
```suggestion
                border: Border(
                  bottom: BorderSide(color: Color(0xFFBDBDBD),
                  width: 0.0,
                ),
              ),
            ),
```